pgconn/pgconn.go

```diff
@@ -1713,6 +1793,8 @@ func Construct(hc *HijackedConn) (*PgConn, error) {
 	}
 
 	pgConn.contextWatcher = newContextWatcher(pgConn.conn)
+	pgConn.bgReader = bgreader.New(pgConn.conn)
+	pgConn.slowWriteTimer = time.AfterFunc(time.Duration(math.MaxInt64), pgConn.bgReader.Start)
```

Comment:
Should this also need something like this?

```
pgConn.frontend = config.BuildFrontend(pgConn.bgReader, pgConn.conn)
```

Reply:
Yes, you are correct. Fixed in cd46cdd450a2c447d50bf30d6a5be513a0631fdf.

Reply:
:+1: